src/fragments/lib/logging/ios/escapehatch/escapehatch.mdx

```diff
@@ -0,0 +1,13 @@
+Add import statements
```

Comment:
can we add a bit more details on why I would use the escape hatch here?

Reply:
Added intro to section with a callout that Swift SDK is still beta to be consistent with other escape hatch pages.